components/reports/wildfire/ReportFlammabilityMap.vue

```diff
@@ -0,0 +1,150 @@
+<template>
+  <div class="has-text-centered has-text-weight-bold">
+    <span v-html="title"></span>
+    <div :id="mapID" class="flammability-minimap"></div>
+  </div>
+</template>
+
+<style lang="scss" scoped>
+.flammability-minimap {
+  height: 15vw;
+  width: 100%;
+}
+</style>
+
+<script>
+import _ from 'lodash'
+import { mapGetters } from 'vuex'
+
+let models = [
+  '5modelAvg',
+  'GFDL-CM3',
```

Comment:
Adjust these to be `[modeling group] [model]` instead of `[modeling group]-[model]`